src/inject/dynamic-theme/stylesheet-modifier.ts

```diff
@@ -30,8 +30,11 @@ export function createStyleSheetModifier() {
     const rulesModCache = new Map<string, ModifiableCSSRule>();
     const varTypeChangeCleaners = new Set<() => void>();
     let prevFilterKey: string = null;
-    let foundNonLoadedLinks = false;
+    // 'false' It means Dark Reader didn't find any non loaded link(s) on this stylesheet.
+    // 'true' It means that Dark Reader has found non loaded link(s).
+    let hasNonLoadedLink = false;
 
+    let hasRebuilt = false;
```

Comment:
`didRebuild` or `wasRebuilt`.

Worth adding a new line before the interface declaration.